browser/ui/views/location_bar/brave_location_bar_view.h

```diff
@@ -8,6 +8,7 @@
 
 #include <vector>
 
+#include "brave/components/ipfs/buildflags/buildflags.h"
```

Comment:
looks like we're missing a dep for `//brave/components/ipfs/buildflags` in `//brave/browser/ui` (it currently has check_includes = false because of some other unresolved issues)